### DISCUSSION TITLE
是否可以吧carpet mod添加成paper服务器之类的插件，以便插件服务器可以使用（Is it possible to add carpet mod as a plugin like paper server so that the plugin server can use it）

### DISCUSSION THREAD

**Discussion by xubinbingood** [gnembon/fabric-carpet#1457] [Q&A]
我们服务器为一个小小的红石服务器，但为了防止玩家搞破坏，所以我们加了CoreProtect插件，但是服务器需要一些假人来进行挂机，我找了很多现在的假人插件，但大多数都在1.18.1版本不可用，而且像Cardboard之类的允许fabric 和插件的mod在1.18.1有严重bug（附魔书属性消失），所以我希望carpet作者可以添加有关carpet的插件，感谢！

Our server is a small redstone server, but in order to prevent players from damaging it, we added the CoreProtect plugin, but the server needs some dummies to hang up. I found many current dummies, but most of them are not available in 1.18.1. And mods like Cardboard that allow fabric and plug-ins were seriously bugged in 1.18.1 (the entrainment book attribute disappeared), so I wish carpet authors would add plug-ins for carpet, thanks!

**Comment by altrisi** [comment]
I'd recommend you to use Ledger for a native alternative to CoreProtect on Fabric, you shouldn't use cardboard and running plugins on Fabric isn't something that's going to work properly unless there's _a lot_ of effort and complexity added.

**Reply by xubinbingood** [reply]
> 我建议你使用 Ledger 作为 CoreProtect on Fabric 的原生替代品，你不应该使用 cardboard 并且在 Fabric 上运行插件不会正常工作，除非增加_了很多_工作量和复杂性。

I would be very grateful if you could provide some information about Ledger

**Reply by altrisi** [reply]
Well, it's a mod with a similar purpose to coreprotect: It logs what happens.

It'd be better if you checked their CurseForge/Modrinth pages.
